README.md

```diff
@@ -4,7 +4,18 @@
 [![Docker](https://img.shields.io/badge/Docker%20Hub-stefanprodan%2Fkube--tools-blue)](https://hub.docker.com/r/stefanprodan/kube-tools)
 
 Github Action with Kubernetes tools:
-kubectl, kustomize, helm, kubeval, conftest, kubeconform, jq, yq, go.
+
+* kubectl
+* yq
+* kustomize
+* helm
+* kubeseal
+* kubeval
+* conftest
+* kubeaudit
+* kubeconform
+* kyverno
```

Comment:
```suggestion
* [kubectl](https://github.com/kubernetes/kubectl)
* [yq](https://github.com/mikefarah/yq)
* [kustomize](https://github.com/kubernetes-sigs/kustomize)
* [helm](https://github.com/helm/helm)
* [kubeseal](https://github.com/bitnami-labs/sealed-secrets)
* [kubeval](https://github.com/instrumenta/kubeval)
* [conftest](https://github.com/open-policy-agent/conftest)
* [kubeaudit](https://github.com/Shopify/kubeaudit)
* [kubeconform](https://github.com/yannh/kubeconform)
* [kyverno](https://github.com/kyverno/kyverno)
```
Added links to the source repos. Helpful when selecting which version to input into the action.